docs/documentation/maintenance-coordinator.md

```diff
@@ -0,0 +1,87 @@
+#Coordinator Model
+
+## Table of Contents
+
+<!-- MarkdownTOC depth=0 autolink="true" bracket="round" autoanchor="true" style="ordered" indent="   " -->
+1. [Role of the Coordinator](#role-of-a-coordinator)
+1. [Change Management](#change-management)
+1. [Issue Management](#issue-management)
+1. [Scenarios](#scenarios)
+1. [What can be expected of a coordinator?](#expectations)
+
+
+<!-- /MarkdownTOC -->
+<a name="role-of-a-coordinator"></a>
+## Role of the Coordinator
+
+The Coordinator is not the owner of the component but much more the curator. 
+The coordinator ensures the quality of contributations to the component 
+and makes it possible for others to commit. The coordinator is responsible 
+that the documentation is kept up to date by the contributors and that the 
+guidelines of the component are met. The coordinator moderates the discussion on 
+finding a vision and on the development. The coordinator further is contact 
+for any sort of question that may arise about the respective component.
+
+The motivation of the coordinator is mostly driven by the need of a 
+reliable component for a certain aspect. Further the coordinator is 
+probably the most attractive contractor for clients aiming to change 
+aspects of the component due to the very indepth know how and the 
+listing as coordinator.
+
+<a name="change-management"></a>
+##Change Management
+Everybody may contribute to any aspect of the component. Such contributions 
+are handed in by pull requests or some other source of data if declared so in the components guidelines. 
+Pull requests on the public interface must be accepted by the JF. The coordinator gives a 
+recommendation to the JF on whether to accept or decline the PR. The decision of the JF may 
+be implicit if no objections to the recommendation of the coordinator is made. If no agreement 
+is achieved in the JF, the Technical Board will decide upon the request.
+
+Final implementations without further changes on the interface do not need 
+formal approval by the JF. The merge of the implementation is performed 
+by the coordinator.
+
```

Comment:
I still think that the relationship to the feature request process needs to be clarified here.

Reply:
Any proposal for such a clarification?

Reply:
Matthias wrote these lines:
https://www.ilias.de/docu/goto.php?target=wiki_5307&client_id=docu#ilPageTocA220
which might need clarifications, too. It will be difficult to do this perfectly now. For me a reference to the FR process and a statement that it needs to respected, too, would be a good start.

Reply:
I'm fine with adding "respect FR process".

The linked definition to me is highly problematic. It touches questions discussed several time (mainly "What may be committed when?") but answers it super restrictive and impracticable.

Reply:
I put that on the list for the next meeting. Best we agree on that, before I make this public.

Reply:
TB: The process for feature requests must be respected.

Note: Should be linked to the respective description in near feature however this description is not available yet.